engine/src/main/scala/cromwell/engine/io/gcs/GcsBatchFlow.scala

```diff
@@ -136,10 +136,10 @@ class GcsBatchFlow(batchSize: Int, scheduler: Scheduler, onRetry: IoCommandConte
     logger.info(s"Going to execute GCS Batch request $batchCommandsHash for the following commands: ${batchCommandNamesList.mkString("\n")}")
     Try(batchRequest.execute()) match {
       case Failure(failure: IOException) =>
-        logger.info(s"Failed to execute GCS Batch request $batchCommandsHash")
+        logger.info(s"Failed to execute GCS Batch request $batchCommandsHash: ${failure.toString}")
```

Comment:
`${failure.toString}` can be shortened to `$failure` here and below. Otherwise LGTM